app/queue.py

```diff
@@ -22,6 +22,9 @@ class Buffer(Enum):
     def name(self, suffix=None):
         return f"{self.value}:{suffix}" if suffix else self.value
 
+    def get_inflight_name(self, receipt: UUID = uuid4(), suffix: str = None) -> str:
+        return f"{Buffer.IN_FLIGHT.value}:{str(suffix)}{str(receipt)}"
```

Comment:
if `suffix` is `None` then the String "None" will be part of the name - is that what we want?

Reply:
Can we separate each value with a `:`, big data design style so we can clearly see each different segment?